CMakeLists.txt

```diff
@@ -385,6 +385,11 @@ if (BUILD_TESTS)
     target_link_libraries (bench ${fftw3_lib})
   endif ()
 
+  if (ENABLE_OPENMP)
+    target_link_libraries (bench ${fftw3_lib}_omp)
+  else ()
+    target_link_libraries (bench ${fftw3_lib})
```

Comment:
I don't think this `else` clause is needed, since `${fftw3_lib}` should have already been added by the preceding `if/else`.